sysutils/rawhide/files/patch-Makefile.diff

```diff
@@ -0,0 +1,17 @@
+Do not overwrite MacPorts's CFLAGS.
+--- Makefile
++++ Makefile
+@@ -135,11 +135,11 @@ RAWHIDE_DEFINES = \
+ #UBSAN_CFLAGS = -fsanitize=undefined
+ #UBSAN_LDFLAGS = -fsanitize=undefined
+ 
+-CC = cc
++#CC = cc
+ #CC = gcc
+ #CC = other
+ CPPFLAGS = $(RAWHIDE_DEFINES) $(PCRE2_DEFINES) $(ACL_DEFINES) $(EA_DEFINES) $(ATTR_DEFINES) $(FLAG_DEFINES) $(MAGIC_DEFINES) $(DEBUG_DEFINES) $(HAVE_SYS_SYSMACROS) $(HAVE_SYS_MKDEV)
+-ALL_CFLAGS = $(CFLAGS) -O3 -g -Wall -pedantic $(CPPFLAGS) $(PCRE2_CFLAGS) $(ACL_CFLAGS) $(EA_CFLAGS) $(ATTR_CFLAGS) $(FLAG_CFLAGS) $(MAGIC_CFLAGS) $(GCOV_CFLAGS) $(UBSAN_CFLAGS)
++ALL_CFLAGS = $(CFLAGS) $(CPPFLAGS) $(PCRE2_CFLAGS) $(ACL_CFLAGS) $(EA_CFLAGS) $(ATTR_CFLAGS) $(FLAG_CFLAGS) $(MAGIC_CFLAGS) $(GCOV_CFLAGS) $(UBSAN_CFLAGS)
```

Comment:
I agree that we want to use our optimization flags (which are within `CFLAGS`) rather than `-O3`, however you probably don't want to remove the rest of the flags the makefile is using here (`-g -Wall -pedantic`). You can just change the order: change `$(CFLAGS) -O3 -g -Wall -pedantic` to `-O3 -g -Wall -pedantic $(CFLAGS)`.

Reply:
I was wondering if that would be a better ordering.